packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -925,204 +925,186 @@ describe("Runtime", () => {
 			const addPendingMessage = (pendingStateManager: PendingStateManager): void =>
 				pendingStateManager.onFlushBatch([{ referenceSequenceNumber: 0 }], 1);
 
-			it(
-				`No progress for ${maxReconnects} connection state changes, with pending state, should ` +
-					"generate telemetry event and throw an error that closes the container",
-				async () => {
-					const pendingStateManager = getMockPendingStateManager();
-					patchRuntime(pendingStateManager);
-
-					for (let i = 0; i < maxReconnects; i++) {
-						addPendingMessage(pendingStateManager);
-						toggleConnection(containerRuntime, i);
-					}
+			it(`No progress for ${maxReconnects} connection state changes, with pending state, should ` +
+				"generate telemetry event and throw an error that closes the container", async () => {
+				const pendingStateManager = getMockPendingStateManager();
```

Comment:
again, a line starting with contents of a function at the same indent as a line before which is part of something else.